### ISSUE TITLE
Screen Reader: Repetitive alt text in footer

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#685]
Problem: the alt text in the universal footer for twitter and youtube are both labelled as links. This causes the screenreader to repeat "link link to mathshare youtube" and "link link to mathshare twitter" 

Solution: the word "link" should be removed from the alt text for the youtube and twitter links in the bottom (dark grey) universal footer
